lib/ddtrace/contrib/faraday/middleware.rb

```diff
@@ -72,6 +72,10 @@ def service_name(env)
           options[:service_name]
         end
 
+        def resource_name(env)
```

Comment:
@giancarlocosta I think this seems totally fine: what's the motivation for doing so? I imagine you want to patch/override this to customize resource names?

Reply:
@delner Yeah that's the goal! 👍

Reply:
@giancarlocosta Okay, ff that's the case, I think that's totally fine, but keep in mind that this component is not considered a part of the public API and therefore might be subject to change in the future without warning. Just want to make sure that risk is understood and acceptable given such a change might break your instrumentation.